docs/usage.md

```diff
@@ -930,9 +930,11 @@ nextflow run nf-core/sarek \
 
 These params can be specified in a config file or in a profile using the params scope, or even in a json or a yaml file using the `-params-file` nextflow option.
 
+Note: we only recommend to store annotation data on S3 if you only store a specific version of the cache for a species, and not the whole cache.
```

Comment:
sarek handles it

Reply:
yeah no, I know why I didn't push for it, it's a mess to do it with the current modules.

Reply:
yeah so handling this is a mess since both vep and snpeff need the path to the root folder.
Which is why I added this extra layer of folder in annotation-cache.